addons/l10n_bd/data/template/account.tax-bd.csv

```diff
@@ -0,0 +1,65 @@
+"id","sequence","description","name","invoice_label","amount","amount_type","type_tax_use","tax_group_id","repartition_line_ids/factor_percent","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/tag_ids","repartition_line_ids/account_id"
```

Comment:
btw, you can remove the column `"repartition_line_ids/factor_percent"` and all the `"100" below, as it is the default